spec/system/login_spec.rb

```diff
@@ -30,18 +38,45 @@
       expect(page).to have_css(".header-dropdown-toggle.current-user")
     end
 
-    it "can login" do
-      # TODO: add test
-    end
-
     it "can reset password" do
-      # TODO: add test
+      Jobs.run_immediately!
+
+      login_modal.open
+      login_modal.fill_username("john")
+      login_modal.forgot_password
+      find("button.forgot-password-reset").click
+
+      wait_for(timeout: 5) { ActionMailer::Base.deliveries.count != 0 }
+
+      mail = ActionMailer::Base.deliveries.last
+      expect(mail.to).to contain_exactly(user.email)
+      reset_password_link = mail.body.to_s[%r{/u/password-reset/\S+}, 0]
+      visit reset_password_link
+
+      find("#new-account-password").fill_in(with: "newsuperpassword")
+      find("form .btn-primary").click
+      expect(page).to have_css(".header-dropdown-toggle.current-user")
     end
   end
 
-  context "with magic link" do
+  context "with login link" do
     it "can login" do
-      # TODO: add test
+      Jobs.run_immediately!
```

Comment:
Might be useful to move `Jobs.run_immediately!` to a before block. It's helpful on all tests in this file.